source/api/content-state/0.3/index.md

```diff
@@ -0,0 +1,675 @@
+
+## 2. Content State
+{: #content-state}
+
+A content state is a JSON-LD data structure that uses the models described by the [IIIF Presentation API][prezi-api] and [W3C Web Annotation Data Model][org-w3c-webanno] specifications. The data structure is a description of a resource, or part of a resource. This data structure must allow the client to load the resource required, and present a particular part of the resource to the user. The state might be very simple: for example, a link to a Manifest. A more complex content state would provide more detail about the intended target. The following are all behaviors a passed-in content state might produce in a compatible client: 
+
+* _load Manifest M_
+* _load Manifest M, navigate to Canvas C, and zoom in to the region defined by xywh=X,Y,W,H_
+* _load Manifest M, navigate such that Range R is selected, and start playing the time-based canvas C within Range R at time t=T_
+
+These intentions can be expressed formally as an Annotation that targets the intended resource, or part of the resource. This content state Annotation must be easily _passed_ into web applications, for example, as a query string parameter, or an HTML `data-` attribute.
+
+### 2.1. Annotation Model for Content State
+
+The target of the Annotation is the IIIF resource, using exactly the same patterns as any other IIIF-targeting Annotation that a IIIF viewer might encounter.
+
+The target could be any resource described by the Presentation API, for example, a:
+
+* Manifest
+* Range
+* Canvas
+* spatial or temporal fragment of a Canvas
+* spatial or temporal point on a Canvas
+
+The Annotation must contain enough information about de-referenceable resources to show the content in context. For example, a Canvas is often not enough information for a viewer to show the intended view; the Manifest the Canvas is part of needs to be declared so that the client can load that Manifest first, and then find the Canvas within it.
+
+### 2.2 Form of Annotation
+
+Any Annotation may have one or more [motivations][org-w3c-webanno-motivation], that provide the reason(s) why is was created. For example, `bookmarking`. 
+
+A content state Annotation always has the motivation `contentState`. This motivation is not defined by either the [W3C Web Annotation Data Model][org-w3c-webanno] or the IIIF Presentation API, and is chosen to avoid potential ambiguity when the target of the content state Annotation is itself an Annotation. The content state annotation may also have additional motivations such as `bookmarking`, `identifying` and so on, as defined by the W3C Web Annotation Model, but it is its particular `contentState` motivation that produces the required behavior in compatible software.
+
+This section shows four possible forms the content state may take, all of which are equivalent to the following example, which is the simplest content state - a reference to a IIIF Manifest.
```

Comment:
The content state _SHOULD_ be one of the following 4, otherwise clients are unlikely to understand it.